eva_submission/eload_ingestion.py

```diff
@@ -0,0 +1,111 @@
+import string
+import subprocess
+
+from bs4 import BeautifulSoup
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.mongo_utils import get_mongo_connection_handle
+import requests
+
+from eva_submission.eload_submission import Eload
+
+
+def sanitize(s):
+    """Lowercases and removes punctuation from s, e.g. to make it a legal MongoDB identifier."""
+    return s.lower().translate(str.maketrans('', '', string.punctuation))
+
+
+class EloadIngestion(Eload):
+    config_section = 'ingestion'  # top-level config key
+    all_tasks = ['metadata_load', 'accession', 'variant_load']
```

Comment:
I am a bit confused... My understanding was that these individual steps would be part of the Nextflow pipeline (thereby facilitating resumption of tasks that failed in one of these steps)...

Reply:
If this is similar to the previous scripts, some of the steps will be managed through direct python call (like the metadata-load) and others will be run through nextflow. But, in any case, the user should have the option of only run some of the step of the script.
This said the whole "step" to nextflow are not very well design but I would leave that off the first pass.

Reply:
Right, as Tim said... I modelled this after the brokering PRs where similar functionality seemed useful, perhaps it's a premature configuration addition but I think it's not a bad idea.